pubtools/_quay/push_docker.py

```diff
@@ -617,12 +617,21 @@ def fetch_missing_push_items_digests(self, push_items, target_settings):
                 v2_sch2_cache = {}
                 for tag in tags:
                     for mtype in missing_media_types:
```

Comment:
Can you explain this method to me? If V2S2 manifest wasn't pushed, it tries to get V2S2 manifest from ??somewhere?? But it's also possible that V2S2 manifest is not there? When would such a scenario occur?

Reply:
v2ch2 can be created on fly when requested by user. But only if there's v2ch1 and not list. If there's list, quay takes list as superior format and pull everything from there. So if there's list but amd64 architecture is not in list, you'll get 404